spring-cloud-dataflow-core/src/main/java/org/springframework/cloud/dataflow/core/dsl/AppNode.java

```diff
@@ -121,4 +123,12 @@ public Properties getArgumentsAsProperties() {
 		return props;
 	}
 
+	public void setLongLivedNonStreamApp(boolean b) {
```

Comment:
TODO: Change this to `unmanagedStreamApp` or `unboundStreamApp` instead of LongLivedNonStreamApp.  `unbound` is more accurate since we do set other app properties.